apps/web/components/PageHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import type {PropsWithChildren} from 'react'
+import {Fragment} from 'react'
+import {HelpButton} from './HelpButton'
+
+type MainContentLayoutProps = PropsWithChildren<{
+  title: string[]
+}>
+
+export function PageHeader({title}: MainContentLayoutProps) {
+  return (
+    <header className="flex items-center border-b border-venice-black-300 py-2 px-4">
```

Comment:
@iwasrobbed I made this a bit roomier than the design. Please let me know what you think.
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/3942264/215314702-61e9c265-4fc5-494c-bf9e-c92ac3180fd6.png">

Reply:
Looks good to me, just want to see it deployed since the fonts look a little large on your screen resolution (I have a zoomed out resolution to fit more on my screen, so might need to tweak font sizes)